### PR TITLE
Refactor unit tests to work with CSS Modules

### PR DESCRIPTION
**Pull request by hugoxiong5**
All the tests broke because CSS Modules changes class names.

I added some configuration settings so that Jest recognizes CSS Modules, and refactored the tests to use updated class names.

### CODE REVIEW

**Comment by hugoxiong5** [comment]
[Fix photo module unit tests](https://trello.com/c/QzKwD42s/349-fix-photo-module-unit-tests)

File: client/PhotoModule.test.js [hunk 1]
```diff
@@ -46,23 +48,24 @@ describe('Responsive Breakpoints', () => {
...
   });
 
-  it('should render gallery view at width >= 640px', () => {
-    global.innerWidth = 640;
+  it('should render gallery view at width >= 1024px', () => {
+    global.innerWidth = 1024;
     global.dispatchEvent(new Event('resize'));
     app.update();
-    expect(app.find('#gallery')).toExist();
+    console.log(app.debug());
```
Comment (nathanq95) [inline client/PhotoModule.test.js]: Not sure if the console log is needed here

**Review by nathanq95 — APPROVED** [review]
Everything looks good.  There's just a a random console log in PhotoModule.test.js that I'm not sure is necessary or not.

File: client/PhotoModule.test.js (hunk 1 above)
Comment (hugoxiong5) [reply client/PhotoModule.test.js]: agree, I'll delete it. Nice catch.

**Pull request merged**